docs/src/pages/components/DateField/datefield.example.html

```diff
@@ -1,115 +1,131 @@
 <div class="example--wrapper">
+  <fieldset class="ds-c-fieldset">
```

Comment:
What a nutty diff... these look to be identical so I'm assuming linting converted spaces to tabs or something? Either way, the content looks fine between these two diffs.